latch/resources/workflow.py

```diff
@@ -98,6 +99,11 @@ def decorator(f: Callable):
                 raise click.exceptions.Exit(1)
 
         _inject_metadata(f, metadata)
-        return _workflow(f)
+
+        wf_name_override = os.environ.get("LATCH_WF_NAME_OVERRIDE")
```

Comment:
please add a comment saying this is only used for serializing inside the container - i guarantee ill be confused looking at this 6 months from now if not lol

Reply:
amen lol